hq/test/logic/SecurityGroupDisplayTest.scala

```diff
@@ -0,0 +1,22 @@
+package logic
+
+import model.{ELB, Instance, SGInUse, UnknownUsage}
+import org.scalatest.{FreeSpec, Matchers}
+import SecurityGroupDisplay._
+
+
+class SecurityGroupDisplayTest extends FreeSpec with Matchers {
+  "resourceIcons" - {
+    "returns 2 when there are two instances" in {
```

Comment:
feels like this case isn't needed as the below one covers it.

Reply:
Good point, I might swap it out for a test for when the list is empty